src/pages/Post/components/ListButton.tsx

```diff
@@ -0,0 +1,11 @@
+import { Button, ButtonProps } from '@chakra-ui/react';
+
+const ListButton = ({ ...props }: ButtonProps) => {
+  return (
+    <Button backgroundColor="green01" color="white" size="lg" _hover={{ backgroundColor: 'green03' }} {...props}>
```

Comment:
#37 를 참고하면 프로젝트에서 공통적으로 사용하는 버튼 색상을 colorSchema로 만들었습니다. colorSchema에 기본적으로 backgroundColor, hoverColor, activeColor가 설정되어있어서 따로 지정해주지 않으셔도 됩니다! 밑의 코드처럼 변경해주시면 됩니다.🙂🙂
```suggestion
    <Button colorScheme="primary" size="lg" {...props}>
```